src/datasets/once/once_utils.py

```diff
@@ -303,3 +294,19 @@ def get_frame_ids_for_scene(once: ONCE,
 
     frame_ids = [os.path.basename(file).split('.')[0] for file in raw_frame_files]
     return sorted(frame_ids)
+
+def __apply_transformation_matrix(point_cloud: np.ndarray,

```

Comment:
maybe we can add to geometry utils since it's used in waymo too?

Reply:
Let's do it later when we start tracking.